CONTRIBUTING.md

```diff
@@ -0,0 +1,270 @@
+
+It is recommended that each of your patches fixes one thing. Smaller patches
+are easier to review, and are thus more likely to be accepted and merged, and
+problems are more likely to be picked up during review.
+
+## Reviews
+
+Before your PRs are merged into the main code base, they will be
+reviewed. Anybody can review any PR and leave feedback (in fact, it
+is encouraged).
+
+We use an "acknowledge" system for people to note if they agree, or disagree,
+with a PR. We utilise some automated systems that can spot common
+acknowledge patterns, which include placing any of these **at the beginning of a
+comment line**:
+
+ - LGTM
+ - lgtm
+ - +1
+ - Approve
+
+### Examples
+
+This is an example of a valid "ack":
+
+```
+Excellent work - thanks for your contribution.
+
+lgtm
+```
+
+But the comment below is *not* valid because the magic "lgtm" does not start
+at the beginning of the line:
+
+```
+I love it! Very clean code and great tests. lgtm.
+```
+
+## Contact
+
+The Kata Containers community can be reached through its dedicated IRC
+channels, Slack channels and mailing lists:
+
+* IRC:
+  * Development discussions: `#kata-dev @ freenode.net`.
+  * General discussions: `#kata-general @ freenode.net`.
+
+* [Slack channels](https://katacontainers.slack.com/) ([invite](http://bit.ly/KataSlack)).
+
+* [Mailing lists](http://lists.katacontainers.io/).
+
+### Project maintainers
+
+The Kata Containers project maintainers will be the ones accepting or
+rejecting any PR. They are listed in the `OWNERS` files, and there can be one
+`OWNERS` file per directory.
+
+The `OWNERS` files split maintainership into 2 categories: reviewers and
+approvers. All approvers also belong to the reviewers list and there must be
+at least one approval from one member of each list for a PR to be merged.
+
+Since approvers are also reviewers, they technically can approve a PR without
+getting another reviewer's approval. However, it is their due diligence to
+rely on reviewers and should use their approval power only in very specific
+cases.
+
+## Issue tracking
+
+To report a bug that is not already documented, please open a github issue for
+the repository in question.
+
+If it is unclear which repository you should be raising your query against,
+first, try getting in [contact](#contact) with us. But if in doubt, raise the
+issue [here](https://github.com/kata-containers/community/issues/new) and we
+will help you to handle the query by routing it to the correct area for
+resolution.
+
+## Closing issues
+
+Our tooling requires a `Fixes` comment to be added to atleast one commit in
+the PR, which triggers github to automatically close the issue when the PR
+is merged:
+
+```
+pod: Remove token from Cmd structure
+
+The token and pid data will be hold by the new Process structure and
+they are related to a container.
+
+Fixes #123
+
+Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>
+```
+
+Github will then automatically close that issue when parsing the
```

Comment:
Lines 269-270 suggested rewrite: The issue is automatically closed by GitHub when the [commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) is parsed.